cloudformation/lambda/geowebcache_seeder/build_and_upload_to_S3.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -ex
+rm -rf geowebcache_seeder.zip *.pyc lib
+pip install -r requirements.txt -t ./lib
+touch ./lib/__init__.py
```

Comment:
Should be able to get rid of this now, since lib shouldn't ever be considered a module in its own right.